src/Common/FitOnFhir.Common.Tests/AuthStateServiceTests.cs

```diff
@@ -136,61 +142,34 @@ public void GivenRedirectUrlNotInApprovedList_WhenCreateAuthStateCalled_ThrowsRe
             SetupHttpRequest(ExpectedToken);
             AuthConfiguration.ApprovedRedirectUrls.Returns(new List<string>());
 
-            Assert.Throws<RedirectUrlException>(() => _authStateService.CreateAuthState(Request));
-
-            _logger.Received(1).Log(
-                Arg.Is<LogLevel>(lvl => lvl == LogLevel.Error),
-                Arg.Is<string>(msg =>
-                    msg == "The redirect URL was not found in the list of approved redirect URLs."));
-
-            _logger.DidNotReceive().Log(
-                Arg.Is<LogLevel>(lvl => lvl == LogLevel.Error),
-                Arg.Is<string>(msg =>
-                    msg == $"The required parameter {Constants.RedirectUrlQueryParameter} was not provided in request"));
+            Assert.Throws<AuthStateException>(() => _authStateService.CreateAuthState(Request));
```

Comment:
I think because all exceptions that are thrown are AuthStateExceptions, we are going to need to validate the exception message. The original bug I found was that we would be showing the wrong message to users if they hit a query parameter exception.